tools/make-one-time-file.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import sys
+from TimeFileMaker import *
```

Comment:
Quoting [the python manual](https://docs.python.org/2/tutorial/modules.html#the-module-search-path):
> When a module named `spam` is imported, the interpreter first searches for a built-in module with that name. If not found, it then searches for a file named `spam.py` in a list of directories given by the variable `sys.path`. `sys.path` is initialized from these locations:
> - the directory containing the input script (or the current directory).
> - `PYTHONPATH` (a list of directory names, with the same syntax as the shell variable `PATH`).
> - the installation-dependent default.

Since `TimeFileMaker` is not a built-in module, `TimeFileMaker.py` is installed to the same directory as `make-one-time-file.py`, the installed copy is picked up.  As I understand it, the parenthetical about "the current directory" applies only to running unsaved scripts.